<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddExchangeRateDialog</class>
 <widget class="QDialog" name="AddExchangeRateDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Exchange Rate</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="primaryCurrencyLabel">
       <property name="text">
        <string>Primary currency</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="primaryCurrencyComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="secondaryCurrencyLabel">
       <property name="text">
        <string>Secondary currency</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="secondaryCurrencyComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
